src/commands.c

```diff
@@ -1072,57 +964,7 @@ IMPLEMENT_COMMAND (save)
     }
 }
 #endif
-#endif
-
-#ifndef USE_WEBKIT2
-IMPLEMENT_COMMAND (frame)
```

Comment:
This one may be implementable using some extension code, but not sure. Should be a new issue.